ui/app/src/views/MigrateView.tsx

```diff
@@ -53,6 +55,8 @@ function MigrateView() {
   const dashboardMutation = useCreateDashboardMutation((data) => {
     navigate(`/projects/${data.metadata.project}/dashboards/${data.metadata.name}`);
   });
+  const { exceptionSnackbar } = useSnackbar();
+  const { data, isLoading } = useProjectList({ onError: exceptionSnackbar });
```

Comment:
wouldn't "projects" be a better name than "data" here?

Reply:
data is the default var returned by the useQuery function

Reply:
yeah definitely, I'm just wondering if it's possible because I think there is a sort of binding so the name should be the same than the one returned by `useProjectList`

Reply:
ok my react-noobie fault here, 'thought it was possible to give any name you want here like for useState

Reply:
You can create an alias if you want `const { data: projects, isLoading }` 😄